Mage.Sets/src/mage/cards/a/AKillerAmongUs.java

```diff
@@ -59,11 +59,7 @@ public AKillerAmongUs(UUID ownerId, CardSetInfo setInfo) {
         super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{4}{G}");
 
         // When A Killer Among Us enters the battlefield, create a 1/1 white Human creature token, a 1/1 blue Merfolk creature token, and a 1/1 red Goblin creature token. Then secretly choose Human, Merfolk, or Goblin.
-        Ability ability = new EntersBattlefieldTriggeredAbility(new CreateTokenEffect(new HumanToken()));
-        ability.addEffect(new CreateTokenEffect(new MerfolkToken())
-                .setText(", a 1/1 blue Merfolk creature token"));
-        ability.addEffect(new CreateTokenEffect(new GoblinToken())
-                .setText(", and a 1/1 red Goblin creature token."));
+        Ability ability = new EntersBattlefieldTriggeredAbility(new CreateTokenEffect(new HumanToken()).withAdditionalTokens(new MerfolkToken(), new GoblinToken()));
```

Comment:
Use one list (see below) and builder style (here): create token effect - with token 1 - with token 2.

Reply:
I think I prefer requiring people to always put at least one token in CreateTokenEffect, but I did add a singular `CreateTokenEffect(firstToken).withAdditionalToken(one additional token)` function for the cases where it's just two token types, would it be better to always add one token at a time so that this would be `new CreateTokenEffect(new HumanToken()).withAdditionalToken(new MerfolkToken()).withAdditionalToken(new GoblinToken())`? Seems more efficient to be able to just pass in several tokens in one pass.

Reply:
The alternative, if we really want to go through with making each of the CreateTokenEffect() constructors without having a singular `token`, is that I would make `CreateTokenEffect(List<Token> tokens, ...)` constructors and just do `CreateTokenEffect(Arrays.asList(new HumanToken(), new MerfolkToken(), new GoblinToken()))` here.